src/components/common/deleteResource.jsx

```diff
@@ -93,7 +93,7 @@ export class DeleteResourceModal extends React.Component {
                        </>
                    }>
                 {this.state.dialogError && <ModalError dialogError={this.state.dialogError} dialogErrorDetail={this.state.dialogErrorDetail} />}
-                <DescriptionList className={this.state.dialogError && "pf-u-pt-md"} isHorizontal>
+                <DescriptionList className={this.state.dialogError && "pf-v5-u-pt-md"} isHorizontal>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230630-122011-699fe1a0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)